src/components/Heading/src/Heading.vue

```diff
@@ -160,7 +160,7 @@ export default {
 
 .Heading {
 	margin: 0;
-	color: var(--color-900);
+	color: var(--color-heading);
```

Comment:
We can drop this altogether if we set the heading textcolor in the default theme.